crates/ruff/src/checkers/ast/mod.rs

```diff
@@ -4933,9 +4945,14 @@ impl<'a> Checker<'a> {
                     let binding = &self.ctx.bindings[*index];
 
                     let full_name = match &binding.kind {
-                        BindingKind::Importation(.., full_name) => full_name,
-                        BindingKind::FromImportation(.., full_name) => full_name.as_str(),
-                        BindingKind::SubmoduleImportation(.., full_name) => full_name,
+                        BindingKind::Importation(Importation { full_name, .. }) => full_name,
```

Comment:
Nit. This could be moved to a 'full_name' method on BindingKind to avoid repetition

Reply:
It doesn't exist on all variants though -- so would it return `Option<&str>`?

Reply:
Yeah, it would have to return an Option